cspell.json

```diff
@@ -21,6 +21,7 @@
     "concat",
     "ctrs",
     "cuda",
+    "curr",
```

Comment:
We were using `cur` in other parts of the code.

Reply:
Changed to `curr` to `cur` in the functions I wrote